pkgs/by-name/un/unrar-free/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, autoreconfHook
+, libarchive
+, pkg-config
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "unrar-free";
+  version = "0.1.3";
+
+  src = fetchFromGitLab {
+    owner = "bgermann";
+    repo = "unrar-free";
+    rev = finalAttrs.version;
+    hash = "sha256-pNcbbHFcEzXKGKUg9nLM3NuUCgZFmFjFa4dXmUuuLYo";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+
+  buildInputs = [ libarchive ];
+
+  meta = {
+    description = "Free utility to extract files from RAR archives";
+    longDescription = ''
+      unrar-free is a free software version of the non-free unrar utility.
+
+      This program is a simple command-line front-end to libarchive, and can list
+      and extract RAR archives but also other formats supported by libarchive.
+
+      It does not rival the non-free unrar in terms of features, but
+      special care has been taken to ensure it meets most user's needs.
+    '';
+    homepage = "https://gitlab.com/bgermann/unrar-free";
+    license = lib.licenses.gpl2Plus;
+    mainProgram = "unrar";
+    maintainers = with lib.maintainers; [ thiagokokada ];
+    platforms = lib.platforms.unix;
+    broken = stdenv.isDarwin;
```

Comment:
It doesn't build on BSDs, and even worse, I doubt it will build on Alpine: https://www.gnu.org/software/gnulib/manual/html_node/error_002eh.html

Reply:
Should have put `platforms.linux` there :facepalm:.

I think it is trivial to patch it to support other systems, but maybe just open an issue upstream>

Reply:
> Should have put `platforms.linux` there :facepalm:.

Do you want to open a PR @wegank ?

Reply:
> Should have put `platforms.linux` there 🤦.
> 
> I think it is trivial to patch it to support other systems, but maybe just open an issue upstream?

Indeed, probably just open an issue upstream.

Reply:
https://gitlab.com/bgermann/unrar-free/-/issues/4